wp-content/themes/core/assets/pcss/layout/_variables.pcss

```diff
@@ -12,4 +12,7 @@ body {
 	--grid-margin: var(--wp--custom--spacing--grid-margin);
 	--grid-gutter: var(--wp--custom--spacing--grid-gutter);
 	--grid-max-width: var(--wp--style--global--wide-size);
+
+	/* this should match the "Grid" size defined in theme.json */
+	--grid-max-grid-width: 1680px;
```

Comment:
🔴  could we differentiate the variable names better between `--grid-max-width` and `--grid-max-grid-width`? Maybe it makes the most sense to rename `--grid-max-width` to `--grid-wide-width` or something?